src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -140,6 +146,30 @@ class FormPage extends React.Component {
       }
     }
 
+    // Bypass the SchemaForm and render the custom component
+    // NOTE: I don't think FormPage is rendered on the review page, so I believe
+    // onReviewPage will always be false here
+    if (typeof route.pageConfig.CustomPage === 'function') {
```

Comment:
I think we have a suitable helper for this that catches some edge cases:

https://github.com/department-of-veterans-affairs/vets-website/blob/4f4d329f489626cc650c3a1935616340c5ad5daa/src/platform/utilities/ui/index.js#L89-L96

Reply:
Oh hey! Nice find! That's much nicer.